udata_gouvfr/theme/templates/home.html

```diff
@@ -62,6 +62,45 @@ <h3>
             </div>
             {# End last reuses #}
 
+            {# Last inventories #}
+            <div class="col-xs-12">
+                <h3>
+                    <a href="{{ url_for('gouvfr.show_page', slug='donnees-cles-par-sujet') }}"
+                        title="{{ _('Données clés par sujet') }}">
```

Comment:
No translation or English string (I vote no translation for this one). Same below.